src/game/GamePlay.hxx

```diff
@@ -14,6 +15,7 @@ public:
 
 private:
   ZoneManager m_ZoneManager;
+  PowerManager m_PowerManager;
```

Comment:
I think we should also add getPowerManager

Reply:
You don’t need an instance of power manager. This is all handled via signals